### DISCUSSION TITLE
[Bug] Weird Spacing When Export to Vanilla

### DISCUSSION THREAD

**Discussion by yassarhanif** [webstudio-is/webstudio-community#199] [Q&A]
I exported my site to vanilla, then upload the static files to cloudflare page. Weirdly the spacing messed up. I encountered this in the last 3 days. It was fine weeks ago.

**Comment by TrySound** [comment]
Please share the link to your site and project

**Reply by yassarhanif** [reply]
https://mofu-hf-bmbtt.wstd.io/

https://maduvitagerd.pages.dev/official-lpwa/
